### DISCUSSION TITLE
[feature request] screenshots / export of blocks + block commenting

### DISCUSSION THREAD

**Discussion by southlakefll** [pybricks/support#1348] [Ideas]
Request for featurings in the future:

It would be nice if there was a built in way to export a clean image of the block coding (like a .png without the grid). There is a way to do this in the LEGO software (but it outputs an svg file which is not super easy to work with, and doesn't output the comments very well).

Also, would be nice if there was a way to add comments on the block coding side.

Both of these features would be very helpful for kids trying to make presentations for FLL.

Thanks!

**Comment by laurensvalk** [comment]
Great suggestions! We'll open two separate issues to track these additions for a future release.

**Comment by ggramlich** [comment]
I just came to the pybricks/discussion to write a similar request and saw this one.

I am looking for a way to write exercise sheets or other documentation for Block programs without having to take screenshots. So I wanted to ask, if there is something similar to https://github.com/scratchblocks/scratchblocks where you can write the program in textual presentation and generate the image from it (svg and/or png would be fine).
